index.js

```diff
@@ -29,9 +30,9 @@ Object.assign(module.exports, {
   disableNetConnect,
   enableNetConnect,
   // TODO-12.x Historically `nock.cleanAll()` has returned the nock global.
-  // The other global methods do noto do this, so it's not clear this was
+  // The other global methods do not do this, so it's not clear this was
   // deliberate or is even helpful. This shim is included for backward
-  // compatibility and shoulud be replaced with an alias to `removeAll()`.
+  // compatibility and should be replaced with an alias to `removeAll()`.
```

Comment:
Can I blame my MacBook keyboard for this? 😆Thanks for the fix.

Reply:
I'm in the same boat. Everything I need an "f" there's a 50/50 chance my laptop will explode.